tests.go

```diff
@@ -260,14 +262,16 @@ func (c CheckOutput) Check(dt string, p string) (retErr error) {
 			return &CheckOutputError{Kind: "contains", Expected: contains, Actual: dt, Path: p}
 		}
 	}
-	if c.Matches != "" {
-		regexp, err := regexp.Compile(c.Matches)
-		if err != nil {
-			return err
-		}
-
-		if !regexp.Match([]byte(dt)) {
-			return &CheckOutputError{Kind: "matches", Expected: c.Matches, Actual: dt, Path: p}
+	for _, matches := range c.Matches {
+		if matches != "" {
```

Comment:
I think we should remove this check here since this would mean someone gave an invalid input.